config/database.yml

```diff
@@ -20,6 +20,8 @@ default: &default
   # For details on connection pooling, see Rails configuration guide
   # https://guides.rubyonrails.org/configuring.html#database-pooling
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  username: postgres
+  password: PassWord
```

Comment:
- [ ] [OPTIONAL] I suggest the use of environment variables to store sensitive data. For example you can make use of the rails inbuilt credentials - `<%= Rails.application.credentials.username %>` then store the values in the encrypted credentials file. To open it use `rails credentials:edit` in the terminal